pkg/tfvars/baremetal/baremetal.go

```diff
@@ -156,6 +162,22 @@ func TFVars(libvirtURI, bootstrapProvisioningIP, bootstrapOSImage, externalBridg
 			})
 	}
 
+	var masterIgn igntypes.Config
+	if err := json.Unmarshal([]byte(ignition), &masterIgn); err != nil {
+		return nil, err
+	}
+	ignitionURL := *masterIgn.Ignition.Config.Merge[0].Source
+	ignitionURLCACert := strings.TrimPrefix(
+		*masterIgn.Ignition.Security.TLS.CertificateAuthorities[0].Source,
+		"data:text/plain;charset=utf-8;base64,")
```

Comment:
These two slice accesses should check to make sure those elements exist first and return an error if not.

Reply:
Added a check to ensure they aren't zero length